content/en/blog/2020/on-demand-egress-gateways/index.md

```diff
@@ -0,0 +1,500 @@
+{{< /text >}}
+
+{{< text bash >}}
+$ kubectl apply -f <my-istiooperator-manifest>
+{{< /text >}}
+
+Where `<my-istiooperator-manifest>` is the path to the a file with following content:
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+  name: istio-operator
+  namespace: istio-operator
+spec:
+  profile: default
+  tag: 1.8.0
+  meshConfig:
+    accessLogFile: /dev/stdout
+    outboundTrafficPolicy:
+      mode: REGISTRY_ONLY
+{{< /text >}}
+
+{{< tip >}}
+Notice that `outboundTrafficPolicy.mode: REGISTRY_ONLY` to block all communications which are not specified by a `ServiceEntry` resource.
+{{< /tip >}}
+
+### Deploy Egress Gateway
+
+The steps for this task assume:
+
+- The service is installed under the namespace: `httpbin`.
+- The service name is: `http-egress`.
+
+As it is mentioned before, `Istio 1.8.0` introduced the possibility to fine-grain control the `Istio` resources. Now, you will take advantage of it.
+
+Create a file with following content:
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  profile: empty
+  tag: 1.8.0
+  namespace: httpbin
+  components:
+    egressGateways:
+    - name: httpbin-egress
+      enabled: true
+      label:
+        app: istio-egressgateway
+        istio: egressgateway
+        custom-egress: httpbin-egress
+      k8s:
+        overlays:
+        - kind: Deployment
+          name: httpbin-egress
+          patches:
+          - path: spec.template.spec.containers[0].readinessProbe
+            value:
+              failureThreshold: 30
+              exec:
+                command:
+                  - /bin/sh
+                  - -c
+                  - curl http://localhost:15021/healthz/ready && curl https://httpbin.org/status/200
+              initialDelaySeconds: 1
+              periodSeconds: 2
+              successThreshold: 1
+              timeoutSeconds: 1
+  values:
+    gateways:
+      istio-egressgateway:
+        runAsRoot: true
+{{< /text >}}
+
+{{< tip >}}
+Notice the block under `overlays`. You are patching the default `egressgateway` to deploy only that component with the new `readinessProbe`
+{{< /tip >}}
+
+Create a `namespace` where you will install the `Egress Gateway`
+
+{{< tip >}}
+Remember that we assume *namespace = httpbin*)
+{{< /tip >}}
+
+{{< text bash >}}
+$ kubectl create ns httpbin
+{{< /text >}}
+
+As it is described in the [documentation](/docs/setup/install/istioctl/#customize-kubernetes-settings), you can deploy several `IstioOperator` resources. However, they have to be pre-parsed and then applied to the cluster.
+
+{{< text bash >}}
+$ istioctl manifest generate -f <path-to-egress-file> | kubectl apply -f -
+{{< /text >}}
+
+Where `<path-to-egress-file>` is the path where you have saved the new `IstioOperator` CR.
+
+### Istio configuration
+
+Now you will set up `Istio` to connect to the external service [https://httpbin.org](https://httpbin.org)
```

Comment:
```suggestion
Now you will configure Istio to allow connections to the external service at [https://httpbin.org](https://httpbin.org).
```

(I'm assuming you're talking about the hosted version, not a locally running httpbin pod)